src/backend/EdFi.DataManagementService.Backend.Test.Unit/PartitionedRepositoryTests.cs

```diff
@@ -51,7 +52,8 @@ public void Setup()
             public int All_possible_last_bytes_return_correct_partition(char lastByte)
             {
                 return _testPartitionedRepository.GetPartition(
-                        new DocumentUuid(_randomDocumentUuid.Substring(0, 35) + lastByte));
+                    (new { Value = _randomDocumentUuid[..35] + lastByte }).ActLike<IDocumentUuid>()
```

Comment:
Using this "ImpromptuInterface" 3rd party package in this test. A small downside of only making internal Core objects available externally as interfaces is that they would be unavailable for plugins to easily instantiate for testing. This is a solution with a nice syntax, especially compared to the mock library.